github-review.el

```diff
@@ -626,6 +589,8 @@ See ‘github-review-start’ for more information"
         (let* ((diff (-> x (elt 0)))
                (pr-object (-> x (elt 1)))
                (comms (github-review-a-get pr-object 'comments))
+	       (commit-id (github-review-a-get (github-review-a-get pr-object 'head) 'sha))
+	       (parent-id (github-review-a-get (github-review-a-get pr-object 'base) 'sha))
```

Comment:
BASE and HEAD refs are found  differently with graphql.

Reply:
Self-Reply 1 by emoonrain starts a review.

Reply:
My first reply to a plain code-block comment.

Reply:
> BASE and HEAD refs are found differently with graphql.

Replied in a while, to expect
1.  ReplyTo == 950695965; 
2.  This comment to be appended to 950697543.

Reply:
> Self-Reply 1 by emoonrain starts a review.

A reply but expectedly a part of Another 2nd top-level review.